SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/statements/CreateRelationStatement.java

```diff
@@ -114,4 +118,18 @@ public String toString() {
                 ", columns=" + this.columns +
                 '}';
     }
+
+    public JsonNode getDefinedObjectSchema() {
+        ObjectMapper mapper = new ObjectMapper();
+        ObjectNode result = mapper.createObjectNode();
+        result.put("name", this.tableName);
```

Comment:
Will this later be usable to also get view definitions too? If so, it'd be good to specify whether something is a table or a view.